packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -23,22 +27,55 @@ const getDocs: () => JSX.Element = () => {
     <>
       <Title>VideoGallery</Title>
       <Description>
-        VideoGallery represents a layout of video tiles for a specific call. It displays a
-        [VideoTile](./?path=/docs/ui-components-videotile--video-tile) for the local user as well as for each remote
-        participant who has joined the call.
+        VideoGallery lays out the local user and each remote participant in a call in a
+        [VideoTile](./?path=/docs/ui-components-videotile--video-tile) component. The VideoGallery component is made up
+        of a [Grid Layout](./?path=/docs/ui-components-videogallery--video-gallery#grid-layout), [Horizontal
+        Gallery](./?path=/docs/ui-components-videogallery--video-gallery#grid-layout), and the [Local Video
+        Tile](./?path=/docs/ui-components-videogallery--video-gallery#local-video-tile). The logic used for which
+        section each [VideoTile](./?path=/docs/ui-components-videotile--video-tile) component is placed is explained
+        below.
       </Description>
 
       <Heading>Importing</Heading>
       <Source code={importStatement} />
 
-      <Heading>Default Example</Heading>
+      <Heading>Grid Layout</Heading>
       <Description>
-        VideoGallery is by default a grid of [VideoTile](./?path=/docs/ui-components-videotile--video-tile) components
-        representing each participant to the call.
+        If there are no remote video streams on, all participants are placed in the [Grid
+        Layout](./?path=/docs/ui-components-gridlayout--grid-layout) including the local user. Otherwise, only remote
+        participants with their video streams on are placed in the Grid Layout. The remaining participants are placed in
```

Comment:
suggestion: Otherwise, only remote participants with their video streams on are placed in the Grid Layout upto a max of `maxRemoteVideoStreams`.